tests/zstreamtest.c

```diff
@@ -1808,7 +1808,8 @@ static int fuzzerTests_newAPI(U32 seed, U32 nbTests, unsigned startTest,
             }
             {   U64 const pledgedSrcSize = (FUZ_rand(&lseed) & 3) ? ZSTD_CONTENTSIZE_UNKNOWN : maxTestSize;
                 ZSTD_compressionParameters cParams = ZSTD_getCParams(cLevel, pledgedSrcSize, dictSize);
-                static const U32 windowLogMax = 24;
+                const U32 windowLogMax = bigTests ? 24 : 20;
```

Comment:
note :
while this will limit the amount of memory when custom compression parameters are passed,
it will not affect tests which select a compression level instead, since it's a different path.

I suspect that memory usage blows up in this other path.

Reply:
It is limited to level 4 in the other path, which is also window log = 20, so it should be okay.